test/integration/dynamic_validation_integration_test.cc

```diff
@@ -117,7 +117,8 @@ TEST_P(DynamicValidationIntegrationTest, LdsFilterRejected) {
   if (reject_unknown_dynamic_fields_) {
     EXPECT_EQ(0, test_server_->counter("listener_manager.lds.update_success")->value());
     // LDS API parsing will reject due to unknown HCM field.
-    EXPECT_EQ(1, test_server_->counter("listener_manager.lds.update_rejected")->value());
+    EXPECT_TRUE(1 == test_server_->counter("listener_manager.lds.update_rejected")->value() ||
```

Comment:
Is this the same case as `integration.cc` below?

Reply:
Done